src/core/src/main/resources/org/apache/jmeter/resources/messages_fr.properties

```diff
@@ -792,6 +792,10 @@ obsolete_test_element=Cet élément de test est obsolète
 once_only_controller_title=Contrôleur Exécution unique
 opcode=Code d'opération
 open=Ouvrir...
+openmodelthreadgroup=Open Model Thread Group
+openmodelthreadgroup_random_seed=Random Seed
+openmodelthreadgroup_schedule_string=Schedule
+openmodelthreadgroup_schedule_summary=Total duration: {0}, max rate: {1} / sec
```

Comment:
@pmouawad , it would be great if you could help with the translations

Reply:
openmodelthreadgroup=Groupe d'unités en modèle ouvert
openmodelthreadgroup_random_seed=Sel pour la fonction aléatoire
openmodelthreadgroup_schedule_string=Planification
openmodelthreadgroup_schedule_summary=Durée totale: {0}, débit max: {1} / sec